vercel/src/lib/AsciiPreview.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import Chip from './Chip.svelte';
+  import type { LanguageColors } from '../types';
+
+  export let name: string;
+  export let ansi: string[];
+  export let hex: string[] | null = null;
+  export let chip: string;
+  export let ascii: string = '';
+
+  let dark = true;
+  let trueColor = hex != null;
+
+  $: html = ascii
+    .split('\n')
+    .map((line) => {
+      // TODO Clean up, this is hard to read
+      let spanCount = 0;
+      const htmlLine = line.replace(/\{(\d+)\}/g, (_match, index) => {
+        const i = Number.parseInt(index, 10);
+        const spanText = trueColor
+          ? `<span style="color: ${hex[i]}">`
+          : `<span class="text-ansi-${ansi[i]}">`;
+        spanCount++;
+        return spanText;
+      });
+      return `${htmlLine}${'</span>'.repeat(spanCount)}`;
+    })
+    .join('\n');
+</script>
+
+<div class="title-row">
+  <!-- TODO Fix a11y warning for form label + control -->
+  <h3><div class="chip"><Chip color={chip} width={24} height={24} /></div>{name}</h3>
```

Comment:
minor: <h3> should not be a container

```suggestion
   <div class="name-with-chip">
    <Chip color={chip} width={24} height={24} />
    <h3>{name}</h3>
  </div>
```

css:
```
.name-with-chip {
    display: flex;
    align-items: baseline;
    gap: 1.5rem;
  }
```

Reply:
You may have missed the css part.

Reply:
You're right, I just added it back.